### PR TITLE
Support windows for gsutil

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Support Windows when using the gsutil step

## Why is it important?

Beats pipelines use it

## Tests

when using the branch itself to test the pipeline -> https://github.com/elastic/apm-pipeline-library/pull/985/commits/0dae25a3d57ac18e67679a772a3cd9ba3906e728

Then `json` files were uploaded correctly within the windows context:

image

image

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -117,13 +117,25 @@ pipeline {
       }
     }
     stage('Google Storage') {
+      steps {
```
Comment (v1v) [inline .ci/Jenkinsfile:120]: Run always

File: .ci/Jenkinsfile [hunk 2]
```diff
@@ -135,9 +147,15 @@ pipeline {
             validateDeclarativePipeline '.ci/Jenkinsfile'
             validateDeclarativePipeline '.ci/buildBeatsDockerImages.groovy'
             validateDeclarativePipeline '.ci/buildDockerImages.groovy'
+            validateDeclarativePipeline '.ci/chatops.groovy'
```
Comment (v1v) [inline .ci/Jenkinsfile:150]: Unrelated

File: vars/gsutil.groovy [hunk 3]
```diff
@@ -38,25 +35,40 @@ def call(Map args = [:]) {
       downloadInstaller(gsUtilLocation)
     }
 
-    withCredentials([file(credentialsId: credentialsId, variable: 'FILE_CREDENTIAL')]) { 
-      sh(label: 'authenticate', script: 'gcloud auth activate-service-account --key-file ${FILE_CREDENTIAL}')
+    withCredentials([file(credentialsId: credentialsId, variable: 'FILE_CREDENTIAL')]) {
+      def credentialsVariable = isUnix() ? '${FILE_CREDENTIAL}' : '%FILE_CREDENTIAL%'
+      cmd(label: 'authenticate', script: 'gcloud auth activate-service-account --key-file ' + credentialsVariable)
```
Comment (v1v) [inline vars/gsutil.groovy:40]: Avoid interpolation for any kind of credentials

**Review by cachedout — APPROVED** [review]
👍
